docs/notebooks/quickrun/qhsri-tutorial.yaml

```diff
@@ -0,0 +1,2 @@
+replace:
+  - { from: "num_steps=8,", to: "num_steps=1," }
```

Comment:
(The advantage of making this a regex (`num_steps=\d+`) is that it would still work if we change the number of steps in the normal run. But since we'll get an error anyway if it doesn't match anything, fine to leave as is.)